modules/utilityFunction.js

```diff
@@ -0,0 +1,121 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export const loadBookslist = (bookInstanceFromBookStore) => {
+  const loopingBook = (book) => `<div id="${book.id}" class="book-card">
+                  <p class="book-details">
+                      "${book.title}" 
+                      by 
+                      ${book.author}
+                  </p>
+                  <button class="remove-button" data-book="${book.id}">Remove</button>
+              </div>`;
+  const bookContainer = document.querySelector('.books-container');
+  bookContainer.innerHTML = bookInstanceFromBookStore.store.map((book) => loopingBook(book)).join('');
+
+  // remove button
+
+  const removeButtons = document.querySelectorAll('.remove-button');
+  removeButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const id = Number(button.getAttribute('data-book'));
+      bookInstanceFromBookStore.remove(id);
+      loadBookslist(bookInstanceFromBookStore);
+      localStorage.setItem('bookCollection', JSON.stringify(bookInstanceFromBookStore.store));
+    });
+  });
+};
+
+export const loadContacts = (contactInstanceFromContact) => {
+  const loopingContacts = (contact) => `<div class="contact-card">
+                <div class="contact-image ${contact.classImage}"></div>
+                <div class="contact-name text-center">${contact.name}</div>
+                <div class="contact-details">
+                  <a class="contact-link" target="_blank" href="${contact.githubLink}">
+                    <i class="fab fa-github-square fa-fw fa-2xl"></i>
+                  </a>
+                  <a class="contact-link" target="_blank" href="${contact.linkedLink}">
+                    <i class="fab fa-linkedin fa-fw fa-2xl"></i>
+                  </a>
+                  <a class="contact-link" target="_blank" href="${contact.instagramLink}">
+                    <i class="fab fa-instagram-square fa-fw fa-2xl"></i>
+                  </a>
+                </div>
+              </div>`;
+  const contactWrapper = document.querySelector('.contact-wrapper');
+
+  contactWrapper.innerHTML = contactInstanceFromContact.profile.map((contact) => loopingContacts(contact)).join('');
+};
+
+export function addBookButton(bookInstanceFromBookStore) {
```

Comment:
- [ ] You are yet to refactor this function to an arrow function. Kindly do that.